pkg/saga/statemachine/engine/core/default_statemachine_config_test.go

```diff
@@ -0,0 +1,202 @@
+package core
+
+import (
+	"errors"
+	"github.com/seata/seata-go/pkg/saga/statemachine/statelang"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"path/filepath"
+	"testing"
+)
+
+func TestDefaultStateMachineConfig_LoadValidJSON(t *testing.T) {
+	config := NewDefaultStateMachineConfig()
+	testFile := filepath.Join("testdata", "order_saga.json")
+
+	err := config.LoadConfig(testFile)
+	assert.NoError(t, err, "加载 JSON 配置应成功")
```

Comment:
Please use English consistently.Please check other places as well and make the necessary changes. Thank you!